code/modules/projectiles/guns/energy.dm

```diff
@@ -10,6 +10,7 @@
 	var/projectile_type = "/obj/item/projectile/beam/practice"
 	var/modifystate
 	var/charge_states = 1 //if the gun changes icon states depending on charge, this is 1. Uses a var so it can be changed easily
+	var/charge_modulo = 25 //Spacing of the charge level sprites
```

Comment:
what did they mean with this var name :thinking:

Reply:
`%`

Reply:
What should I call it?